web/elm/src/Dashboard/Grid/Constants.elm

```diff
@@ -21,17 +21,27 @@ cardBodyHeight =
 
 padding : number
 padding =
-    25
+    32
+
+
+carHeaderVerticalSpace : number
+carHeaderVerticalSpace =
+    8
 
 
 cardHeaderHeight : number -> number
 cardHeaderHeight numRows =
-    20 + 30 * numRows
+    32 + 21 * numRows + (numRows - 1) * carHeaderVerticalSpace
```

Comment:
Can we extract these numbers to named constants, and use the constants when defining the styles in `Dashboard/Styles.elm`? e.g. this line becomes

```
    2 * cardHeaderPadding + cardHeaderRowLineHeight * numRows + (numRows - 1) * cardHeaderRowGap
```

and then in `Dashboard/Styles.elm` we use `cardHeaderPadding` in place of the hard-coded `16px` padding, and `cardHeaderRowLineHeight` in place of the hard-coded `21px` line height. Similarly in `Dashboard/Pipeline.elm`, we can use `cardHeaderRowGap` in place of the hard-coded `8px` height.

This way, there's an explicit dependency between the actual styles and their manually calculated heights, so the two can't drift apart